README.md

```diff
@@ -84,7 +84,7 @@ Now start the server:
 $ ./multinode-demo/leader.sh
 ```
 
-To run a performance-enhanced fullnode on Linux,
+To run a performance-enhanced leader,
```

Comment:
I think "on Linux" is still valid here as we don't currently support the perf libs on any other platform